server/nestjs/src/auth/auth.strategy.ts

```diff
@@ -1,17 +1,26 @@
 import { LoginStrategy } from 'passport-did-auth';
 import { Injectable } from '@nestjs/common';
 import { PassportStrategy } from '@nestjs/passport';
+import * as fs from 'fs';
 
 @Injectable()
 export class AuthStrategy extends PassportStrategy(LoginStrategy, 'login') {
   constructor() {
-    const acceptedRoles = process.env.ACCEPTED_ROLES ? process.env.ACCEPTED_ROLES.split(',') : [];
+    const acceptedRoles = process.env.ACCEPTED_ROLES
+      ? process.env.ACCEPTED_ROLES.split(',')
+      : [];
     super({
-      jwtSecret: 'secret',
-      name: 'login',
-      rpcUrl: 'https://volta-rpc.energyweb.org/',
-      cacheServerUrl: 'https://volta-identitycache.energyweb.org/',
-      acceptedRoles
+      jwtSecret: fs.readFileSync('private.pem'),
+      jwtSignOptions: {
+        algorithm: 'RS256',
+      },
+      rpcUrl: process.env.RPC_URL || 'https://volta-rpc.energyweb.org/',
+      cacheServerUrl:
+        process.env.CACHE_SERVER_URL ||
+        'https://identitycache-dev.energyweb.org/',
```

Comment:
Hmm, not sure it makes sense to change the cacheserver url to dev. Could be weird if the front-end and the back-end are pointing to different cache-servers. And I think it makes sense for the examples to be using the "volta" server. I think we just wait until auth is deployed to the Volta cache server and then deploy these example updates.

Reply:
👍